src/components/Microsoft.DotNet.UpgradeAssistant.Analysis/AnalyzeResultWriter.cs

```diff
@@ -30,8 +30,6 @@ public async Task WriteAsync(IAsyncEnumerable<AnalyzeResultDefinition> results,
 
             var sarifLog = new SarifLog()
             {
-                Version = SarifVersion.Current,
-                SchemaUri = new Uri("http://json.schemastore.org/sarif-2.1.0"),
```

Comment:
When we create a SarifLog, by default it points to the Current version and the correct schema. With that, we don't need to change it.